src/services/db/review.ts

```diff
@@ -83,3 +90,45 @@ export const approvePullRequest = (
       },
     }
   )
+
+export const getComments = async (
+  siteName: string,
+  pullRequestNumber: number
+) => {
+  const rawComments = await axiosInstance
+    .get<RawComment[]>(`${siteName}/issues/${pullRequestNumber}/comments`)
+    .then(({ data }) => data)
+  return _.compact(
+    rawComments.map((rawComment) => {
+      try {
+        const commentData = JSON.parse(rawComment.body)
```

Comment:
think i have a type + util for this - could be in `review`! maybe we should use that instead?

Reply:
Swapped in 2667ea5 - Zhong Jun also raised that we shouldn't be putting users' emails directly in the comment, so that has been modified as well